.jshintrc

```diff
@@ -13,8 +13,7 @@
   "unused": "vars",
 
   "node": true,
-  
+
   "trailing": true,
-  "curly": true,
-  "indent": 2
```

Comment:
I personally prefer the combination of JSHint and JSCS—it's your call, not mine, but I find that the two special-purpose tools complement each other nicely.